sdk/servicebus/service-bus/src/modelsToBeSharedWithEventHubs.ts

```diff
@@ -18,7 +19,18 @@ export type OperationOptionsBase = Pick<OperationOptions, "abortSignal" | "traci
  * @ignore
  */
 export function getParentSpan(
-  options: Pick<OperationOptionsBase, "tracingOptions">
+  options?: OperationTracingOptions
 ): Span | SpanContext | null | undefined {
-  return options.tracingOptions?.spanOptions?.parent;
+  return options?.spanOptions?.parent;
+}
+
+/**
+ * The set of options to manually propagate `Span` context for distributed tracing.
+ * - `parentSpan` : The `Span` or `SpanContext` for the operation to use as a `parent` when creating its own span.
+ */
+export interface TryAddOptions {
+  /**
+   * The `Span` or `SpanContext` to use as the `parent` of any spans created while calling operations that make a request to the service.
+   */
+  parentSpan?: Span | SpanContext | null;
```

Comment:
I noticed in Event Hubs that `null` is not an option for `parentSpan`. Why do we have this here?

Reply:
I'm confused.
There are other places where we allow the span to be "null" which is a valid value and it signifies the rootSpan.

I'll remove the null from here to be consistent and later log an issue to handle my confusion in both event-hubs and service-bus.

Reply:
There are other places where we allow the span to be "null" which is a valid value and it signifies the rootSpan.

Moreover, I needed to allow this "null" since I'm calling getParentSpan() for the tryAdd over array of messages which may return a null value.

In event-hubs, this tracing for "array of messages" case is duplicated.
"null" is a valid value and event-hubs should also allow this.

Reply:
I'll log an issue for fixing it in event-hubs.

Reply:
Logged https://github.com/Azure/azure-sdk-for-js/issues/11687